website_form_recaptcha/controllers/main.py

```diff
@@ -27,7 +27,7 @@ def recaptcha_public(self):
     def extract_data(self, model, values):
         """ Inject ReCaptcha validation into pre-existing data extraction """
         res = super(WebsiteForm, self).extract_data(model, values)
-        if model.website_form_recaptcha:
+        if model.sudo().website_form_recaptcha:
```

Comment:
```suggestion
        if model.with_user(SUPERUSER_ID).website_form_recaptcha:
```

It should be like that?

Reply:
I don't think so :sweat_smile: `sudo` is only replace by `with_user` when the access is made in behalf of such user. That doesn't matter here as there's no more context to give.

Reply:
Yes, `sudo` is the way to do it, as no impersonation is needed, just bypass security rules.